maufbapi/http/base.py

```diff
@@ -196,6 +209,18 @@ def get(
                 return sandboxed_get(url)
         return self.http.get(url, headers=headers, **kwargs)
 
+    def http_get(self, *args, **kwargs) -> ClientResponse:
+        return await self.proxy_with_retry(
+            "AndroidAPI.get",
+            lambda: self.raw_http_get(*args, **kwargs),
+        )
```

Comment:
This one doesn't seem to be used anywhere

Reply:
🤦  https://github.com/mautrix/facebook/pull/294/commits/bb7cb854c9907a7d061e3b0b5ad784d697e440ad